be/src/storage/rowset/column_reader.h

```diff
@@ -150,10 +147,18 @@ class ColumnReader {
     Status bloom_filter(const std::vector<const ::starrocks::vectorized::ColumnPredicate*>& p,
                         vectorized::SparseRange* ranges);
 
-    uint32_t version() const { return _storage_format_version; }
-
     Status load_ordinal_index_once();
 
+    inline const std::string& file_name() const { return _segment->file_name(); }
+
+    inline MemTracker* mem_tracker() const { return _segment->mem_tracker(); }
+
+    inline fs::BlockManager* block_manager() const { return _segment->block_manager(); }
+
+    inline bool keep_in_memory() const { return _segment->keep_in_memory(); }
+
+    inline uint32_t num_rows() const { return _segment->num_rows(); }
```

Comment:
```suggestion
    const std::string& file_name() const { return _segment->file_name(); }

    MemTracker* mem_tracker() const { return _segment->mem_tracker(); }

    fs::BlockManager* block_manager() const { return _segment->block_manager(); }

    bool keep_in_memory() const { return _segment->keep_in_memory(); }

    uint32_t num_rows() const { return _segment->num_rows(); }
```

Reply:
Functions defined within the class body are inline functions implicitly, there is no need to add the inline keyword.